packages/grafana-ui/src/components/FormField/FormField.tsx

```diff
@@ -44,3 +47,15 @@ export const FormField: FunctionComponent<Props> = ({
 
 FormField.displayName = 'FormField';
 FormField.defaultProps = defaultProps;
+
```

Comment:
Instead of using defaultProps you can spread the defaultProps in the Component declaration:
```ts
export const FormField: FunctionComponent<Props> = ({
  label,
  tooltip,
  labelWidth,
  inputWidth,
  inputEl,
  className,
  ...defaultProps,
  ...inputProps
}) => {	}) => {
```

Reply:
@dprokop should we use <Input /> from grafana-ui here instead of a regular <input />. No need for the ternary then is my thinking? Maybe out of scope for this PR as it effects a handfull of places that would need to be tested.
```ts
<input type="text" className={`gf-form-input ${inputWidth ? `width-${inputWidth}` : ''}`} {...inputProps} />
```

Reply:
```
export const FormField: FunctionComponent<Props> = ({
  label,
  tooltip,
  labelWidth,
  inputWidth,
  inputEl,
  className,
  ...defaultProps,
  ...inputProps
}) => {
```
@hugohaggmark It seems like javascript does not support using multiple rest element in a destructuring pattern. I tried the code above and I got this error:
`A rest element must be last in a destructuring pattern.ts(2462)`

Reply:
Oops, I missed that second one, my bad.

Reply:
okidoki 👍